project1/css/index.css

```diff
@@ -64,6 +64,74 @@ h1, h2, h3, h4, h5 {
 	margin-bottom: 10px;
 }
 
-cta {
-	display: inline-block;
+.cta {
+	padding: 10px 50px;
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	justify-content: space-around;
+	border-bottom: 5px solid black;
+}
+
+.cta-text {
+	font-size: 90px;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+.main-content {
+	width: 100%;
+	padding: 5px 20px;
+}
+
+.top-content {
+	font-size: 25px;
+	padding: 10px 40px;
+	border: 10px;
+	margin: 10px 40px;
+	display: flex;
+	justify-content: space-around;
+}
+
+.middle-img {
+	width: 100%;
+}
+
+.bottom-content {
+	font-size: 25px;
+	padding: 10px 40px;
+	border: 10px;
+	margin: 10px 40px;
+	display: flex;
+	justify-content: space-around;
+	border-bottom: 5px solid black;
+}
+
+.contact {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+footer {
+	display: flex;
+	justify-content: center;
+}
+
+.cta-text h1 {
+	display: flex;
+	justify-content: center;
+	flex-flow: column;
+	text-align: center;
+}
+
+button {
+	width: 100px;
+	justify-self: center;
+}
+
+.container {
+	width: 2000px;
 }
```

Comment:
I know we touched on this briefly yesterday. It'd be nice to see the container centered.